src/common/components/select-repository-list/index.js

```diff
@@ -196,9 +184,9 @@ export class SelectRepositoryListComponent extends Component {
               { selectedRepositories.length } selected
             </strong>
             {' '}
-            (<a href="#" onClick={this.onHelpClick.bind(this)}>
+            (<Button appearance={ 'link' } onClick={this.onHelpClick.bind(this)} aria-label={ this.state.showMissingReposInfo ? 'Click to close the missing / private repos dialog' : 'Click to open the missing / private repos dialog' }>
```

Comment:
@bartaz as much as that is true to describe hidden content it should also be used to explain actions when they're not clear. For example a download button is great and could say "Download ubuntu". But for accessibility we would add an aria-label to explain it would be downloading ubuntu, the size of the document and the type of file. This is something we've done in MAAS and I will be trying to do this in BSI.

i'll definitely switch the "/" to or though :)